Sources/UIImagePickerController+Promise.swift

```diff
@@ -5,10 +5,21 @@ import UIKit
 
 #if !os(tvOS)
 
+public struct AnimationOptions: OptionSet {
```

Comment:
Hello @lstomberg 
Why do you need to re-define `PMKAnimationOptions`?

Reply:
Presumably for SwiftPM since it won't get the objective-c portions